### PR TITLE
Add vuid 06559, Validate VK_INSTANCE_CREATE_ENUMERATE_PORTABILITY_BIT_KHR

### PR DESCRIPTION
**Pull request by ziga-lunarg**

### CODE REVIEW

File: layers/parameter_validation_utils.cpp [hunk 1]
```diff
@@ -86,9 +86,18 @@ bool StatelessValidation::validate_instance_extensions(const VkInstanceCreateInf
     InstanceExtensions local_instance_extensions;
     local_instance_extensions.InitFromInstanceCreateInfo(specified_version, pCreateInfo);
 
+    bool has_portability_enumeration = false;
```
Comment (jeremyg-lunarg) [inline layers/parameter_validation_utils.cpp:89]: Can't you do this by looking at` local_instance_extensions.vk_khr_portability_enumeration`?

**Review by ncesario-lunarg — APPROVED** [review]
LGTM!

**Review by jeremyg-lunarg — APPROVED** [review]
LGTM

**Comment by sjfricke** [comment]
Ping, ran into to this today and could have used this VUID to catch it

**Comment by ncesario-lunarg** [comment]
@ziga-lunarg just getting rid of the `ExpectSuccess` and `VerifyNotFound` calls should make this good to go.

**Pull request merged**